include/oneapi/dpl/pstl/glue_algorithm_ranges_impl.h

```diff
@@ -26,7 +26,7 @@
 #endif
 
 #include "utils_ranges.h" // __difference_t
-#include "utils.h"        // oneapi::dpl::__internal::__swap_ranges_fn
+#include "utils.h"        // oneapi::dpl::__internal::__swap_fn
```

Comment:
`__swap_fn` is not used in `glue_algorithm_ranges_impl.h` anymore. Is that inclusion still neede?

Reply:
rather is not. good catch